mmpose/datasets/pipelines/top_down_transform.py

```diff
@@ -22,15 +22,18 @@ class TopDownRandomFlip:
 
     def __init__(self, flip_prob=0.5):
         self.flip_prob = flip_prob
+        # A flag to indicate whether flip is done for children class.
+        self.flip_flag = 0
```

Comment:
Why save this flag to a class? It will be changed after each `__call__`, and how to access its value since the object is mixed in the crowd of pipelines.

Reply:
I want to re use the codes of TopDownRandomFlip for HandRandomFlip. In HandRandomFlip, the flip process of 'img', 'joints_3d', 'joints_3d_visible' and 'center' is the same with TopDownRandomFlip, and we only need to flip the 'hand type' and 'rel_root_depth'.
In want to make HandRandomFlip as a child class of TopDownRandomFlip, and I need a flag to determine whether flip is done or not.

Reply:
I see, makes sense. How about making it a bool?

Reply:
In order to avoid possible bugs with multiple workers, I changed the flag from an attribute of TopDownRandomFlip to an item in results.